test/com/puppetlabs/http/client/impl/metrics_unit_test.clj

```diff
@@ -0,0 +1,150 @@
+(ns com.puppetlabs.http.client.impl.metrics-unit-test
+  (:require [clojure.test :refer :all]
+            [puppetlabs.http.client.async :as async])
+  (:import (com.codahale.metrics MetricRegistry)
+           (com.puppetlabs.http.client.impl Metrics Metrics$MetricType)
+           (org.apache.http.message BasicHttpRequest)))
+
+(def bytes-read Metrics$MetricType/BYTES_READ)
+
+(defn add-metric-ns [string]
+  (str "puppetlabs.http-client.experimental." string))
+
+(deftest start-bytes-read-timers-test
```

Comment:
Should there be `is` statements around each of the `(= ...)` blocks?  Seems like these tests would just be a no-op as currently written?